activesupport/lib/active_support/cache/redis_cache_store.rb

```diff
@@ -365,6 +365,18 @@ def read_multi_entries(names, **options)
           end
         end
 
+        ##
+        # :method: write
+        # :call-seq: write(name, value, options = nil)
+        #
+        # Behaves the same as ActiveSupport::Cache::Store#write, but supports
+        # additional options specific to Redis.
+        #
+        # ==== Additional Options
+        #
+        # * <tt>unless_exist: true</tt> - Prevents overwriting an existing cache
```

Comment:
RedisCacheStore isn't the only one to support that option. `MemCacheStore`, `MemoryStore` and `FileStore` also support it. Hence it should be documented in `AS::Cache#write`.

Reply:
Even better!